### PR TITLE
[WIP] (FM-7785) Additional test cleanup

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

File: spec/acceptance/firewall_attributes_happy_path_spec.rb [hunk 1]
```diff
@@ -185,14 +184,26 @@ class { '::firewall': }
...
-          firewall {
-            '601 - clamp_mss_to_pmtu':
+          firewall { '601 - clamp_mss_to_pmtu':
               proto             => 'tcp',
               chain             => 'FORWARD',
               tcp_flags         => 'SYN,RST SYN',
               jump              => 'TCPMSS',
               clamp_mss_to_pmtu => true,
           }
+          firewall { '601 disallow esp protocol':
```
Comment (tphoney) [inline spec/acceptance/firewall_attributes_happy_path_spec.rb]: can you give this a unique number

File: spec/acceptance/firewall_attributes_happy_path_spec.rb [hunk 2]
```diff
@@ -249,6 +260,122 @@ class { '::firewall': }
             uid => '!0',
             proto => 'all',
           }
+          firewall { '1000 - set_dscp':
```
Comment (tphoney) [inline spec/acceptance/firewall_attributes_happy_path_spec.rb]: keep the numbers in the names in order

Comment (tphoney) [reply spec/acceptance/firewall_attributes_happy_path_spec.rb]: just move them up.

File: spec/acceptance/firewallchain_spec.rb [hunk 3]
```diff
@@ -1,11 +1,6 @@
 require 'spec_helper_acceptance'
 
 describe 'puppet resource firewallchain command' do
-  before :all do
```
Comment (tphoney) [inline spec/acceptance/firewallchain_spec.rb]: this is normally needed for every test, so we can re-run the tests. why was it removed ?

**Comment by tphoney** [comment]
looks good some minor comments.

**Comment by eimlav** [comment]
@tphoney updated according to changes. Also ensured unique rule numbers are used in ipv6 happy path and made names of rules more explicit.

**Comment by tphoney** [comment]
@eimlav looks good, a run through the ad-hoc pipeline and removing the WIP from the title and i would be happy to merge.

**Comment by eimlav** [comment]
Adhoc ran successfully.

Screen Shot 2019-03-14 at 12 43 48

**Comment by eimlav** [comment]
@tphoney If you're happy with the changes, do you want me to rebase or are you happy to just squash it all and remove the messages?

**Pull request merged**